Tests/sdk_release/create_content_pr.py

```diff
@@ -0,0 +1,151 @@
+
+SLACK_CHANGELOG_FILE = 'CHANGELOG_SLACK.txt'
+SLACK_MERGE_PRS_FILE = 'SLACK_MERGE_PRS_REQUEST.txt'
+CONTENT_PR_NUMBER_FILE = 'CONTENT_PR.txt'
+
+TIMEOUT = 60 * 60  # 1 hour
+
+PRS_LIST_TEMPLATE = 'https://api.github.com/repos/demisto/{repo}/pulls'
+UPDATE_SDK_VERSION_WORKFLOW = 'https://api.github.com/repos/demisto/content/actions' \
+                              '/workflows/update-demisto-sdk-version.yml/dispatches'
+
+SLACK_MERGE_PRS_MESSAGE = 'Please merge the demisto-sdk and content pull requests:\n{sdk_pr}\n{content_pr}'
+SLACK_RELEASE_MESSAGE = 'demisto-sdk `{sdk_version}` has been released :party-github:\n' \
+                        ':alert: Please run in the terminal\n' \
+                        '`~/dev/demisto/demisto-sdk/demisto_sdk/scripts/update_demisto_sdk_version.sh ~/dev/' \
+                        'demisto/content ~/dev/demisto/demisto-sdk`\nChange log\n```\n{changelog}\n```'
+
+
+def get_pr_from_branch(repository, branch, access_token):
+    url = PRS_LIST_TEMPLATE.format(repo=repository)
+    params = {'head': f'demisto:{branch}'}
+    res = requests.get(url, headers={'Authorization': f'Bearer {access_token}'},
+                       params=params, verify=False)
+    if res.status_code != requests.codes.ok:
+        logging.error(f'Failed to retrieve pull request from branch {branch}')
+        logging.error(res.text)
+        sys.exit(1)
+
+    prs_list = res.json()
+    if prs_list:
+        return prs_list[0]
+    return None
+
+
+def options_handler():
+    parser = argparse.ArgumentParser(description='Triggers update-demisto-sdk-version workflow')
+
+    parser.add_argument('-t', '--access_token', help='Github access token', required=True)
+    parser.add_argument('-b', '--release_branch_name', help='The name of the release branch', required=True)
+    parser.add_argument('-r', '--reviewer', help='The reviewer of the pull request', required=True)
+    parser.add_argument('--artifacts-folder', help='Artifacts folder to create the files', required=True)
+    parser.add_argument('-d', '--is_draft', help='Is draft pull request', default='FALSE')
+    options = parser.parse_args()
+    return options
+
+
+def main():
+    install_logging("create_content_pr.log", logger=logging)
+
+    options = options_handler()
+    release_branch_name = options.release_branch_name
+    access_token = options.access_token
+    reviewer = options.reviewer
+    artifacts_folder = options.artifacts_folder
+    is_draft = options.is_draft
+
+    if is_draft and bool(strtobool(is_draft)):
+        is_draft = True
+        logging.info('preparing to trigger update-demisto-sdk-version workflow with draft pull request')
+    else:
+        is_draft = False
+        logging.info('preparing to trigger update-demisto-sdk-version workflow')
+
+    # prepare the inputs for trigger update-demisto-sdk-version workflow
+    inputs = {
+        'reviewer': reviewer,
+        'release_version': release_branch_name,
+        # 'is_draft': is_draft,
```

Comment:
why it's commented?

Reply:
because we use this code yesterday for the release and  `update-demisto-sdk-version.yml` in master dont have this input